scripts/README.md

```diff
@@ -3,8 +3,9 @@
 This project depends on some remote sources for data:
 
 - Events: Google Sheets, using [Google Sheets API](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/get)
+- Newsletter: [Mailchimp API](https://mailchimp.com/developer/)
 
-The scripts in this directory are used to fetch the data and make it available to the Vue application to render the new content.
+The scripts in this directory are used to fetch the data and make it available to the Vue application to render the new content, as well as some other side effects.
```

Comment:
I didn't want to be specific in this section here because this is just explaining in a general way what the scripts we have built (and will build) do. In the specific commands docs I explain all of this.